README.md

```diff
@@ -69,6 +69,10 @@ For more information and distro-specific instructions, see the [Linux installati
 
 MSI installers are available for download on the [releases page][].
 
+### Github Actions
+
+`gh` is pre-installed on the `ubuntu-latest` [Github Hosted Runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners)
```

Comment:
Isn't gh preinstalled on all github runners? I'm sure it's available on Windows too; I just haven't checked macOS.

Reply:
I updated the readme in commit https://github.com/cli/cli/pull/3681/commits/9170ae6f97a4058ade7c4c0ef6b09a84fb323d87 to reflect that it's installed on Github Hosted Runners which reflects @vilmibm's comment in https://github.com/cli/cli/issues/2410#issuecomment-782241878